pdns/dnsdist-web.cc

```diff
@@ -531,6 +546,30 @@ static void connectionThread(int sock, ComboAddress remote)
           output << statesbase << "tcpavgconnduration"     << label << " " << state->tcpAvgConnectionDuration   << "\n";
         }
 
+        const string rule_action = "dnsdist_rule_action";
+        output << "# HELP " << rule_action << " Number of queries matching the specified action rule" << "\n";
+        output << "# TYPE " << rule_action << " counter" << "\n";
+        auto actionRules = someResponseRulesToPrometheus(rule_action, &g_rulactions);
+        output << actionRules;
+
+        const string rule_response = "dnsdist_rule_response";
+        output << "# HELP " << rule_response << " Number of queries matching the specified response rule" << "\n";
+        output << "# TYPE " << rule_response << " counter" << "\n";
+        auto responseRules = someResponseRulesToPrometheus(rule_response, &g_resprulactions);
+        output << responseRules;
+
+        const string rule_cacheHit = "dnsdist_rule_cachehit";
+        output << "# HELP " << rule_cacheHit << " Number of queries matching the specified cache hit rule" << "\n";
+        output << "# TYPE " << rule_cacheHit << " counter" << "\n";
+        auto cacheHitResponseRules = someResponseRulesToPrometheus(rule_cacheHit, &g_cachehitresprulactions);
+        output << cacheHitResponseRules;
+
+        const string rule_selfAnsweredResponse = "dnsdist_rule_selfansweredresponse";
```

Comment:
Hi, for the time being (and hopefully going forward), pdns master is running the [check-spelling](https://github.com/marketplace/actions/check-spelling) action. In order to avoid a ❌ when merged https://github.com/jsoref/pdns/commit/6352b9816ec2624646bfc68380f29a90422e36c2#commitcomment-39229536 -- you'll need to merge/rebase on `master` to get the check (and the files) -- you can just accept the advice (which should add `selfansweredresponse` to `expect.txt`) as `selfansweredresponse` appears to be the proper thing here.